docs/quick_start.md

```diff
@@ -1,127 +1,140 @@
-# 🚀 Quick start
+# 🚀 Quick Start
 
 This guide will instruct you in writing and deploying your first _smart function_ in under 10 minutes.
 
-It assumes that you have already [installed `jstz`](installation.md) and have a basic familiarity with [Javascript](https://www.youtube.com/watch?v=lkIFF4maKMU).
+It assumes that you have already [installed `jstz`](installation.md) and have a basic familiarity with [JavaScript](https://www.youtube.com/watch?v=lkIFF4maKMU).
 
 ## What is jstz?
 
-`jstz` is a specialized JavaScript runtime for Tezos smart optimistic rollups that aims to be compatible with web conventions.
+`jstz` is a specialized JavaScript runtime for [Tezos Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) that aims to be compatible with web conventions.
 
 With `jstz` you can deploy so called _smart functions_ which are operating similarly to cloud functions, while running on Tezos L2 and
 providing additional security and blockchain-specific functionality typical for smart contracts.
 
-## 1. Write your smart function
+## Your First Smart Function
 
-Let's see how a smart function looks like with the following example.
-The `request_tez` smart function allows users to request tez from you if asked politely.
+In this example, the smart function takes a HTTP `Request` object and returns a `Response` object
+that sent a tez to the requester if asked politely.
```

Comment:
this sounds a bit complicated to me and don't think it is true that the 'Response object sent tez to the requester'. 
Since the Request/Response handling is described in more detail later, I suggest keeping it simple and just describing what the sf does.

Reply:
I think it's better to mention explicitly the HTTP `Request` / `Response`, but there is a grammatical error at the end of this sentence